apps/aecore/src/aec_conductor.erl

```diff
@@ -51,12 +51,16 @@
 
 %% Mining API
 -export([ get_mining_state/0
-        , get_miner_account_balance/0
-        , next_nonce_for_account/1
         , start_mining/0
         , stop_mining/0
         ]).
 
+%% State trees API
+-export([ get_miner_account_balance/0
+        , next_nonce_for_account/1
+        , get_all_accounts_balances/0
```

Comment:
A test on aec_conductor would be helpful - for the assumption that top always have state trees.